### ISSUE TITLE
Keep CHANGELOG.md and Release Notes in sync

### ISSUE THREAD

**Issue by fschmager** [immutables/immutables#1252]
Dependabot notified me that a new version of `value` was available `2.8.9-ea-1` (which itself is an unfortunate naming but that's already being discussed in #1251).
I've tried to find out what had changed but couldn't find that version under the [Github Releases tab](https://github.com/immutables/immutables/releases). Only `2.8.8` was listed:
<img width="788" alt="Screen Shot 2020-11-30 at 08 35 43" src="https://user-images.githubusercontent.com/10031576/100616422-5ea4f500-3319-11eb-99c6-7df681b7f104.png">

Which lead me to check [CHANGELOG.md](https://github.com/immutables/immutables/blob/master/CHANGELOG.md) but the last entry there is for `2.8.1`:
<img width="493" alt="Screen Shot 2020-11-30 at 08 35 24" src="https://user-images.githubusercontent.com/10031576/100616442-66649980-3319-11eb-9890-105c7d338399.png">

Also noticed that the docs refer to `2.8.2` as the latest release:
<img width="1232" alt="Screen Shot 2020-11-30 at 08 32 12" src="https://user-images.githubusercontent.com/10031576/100616312-361cfb00-3319-11eb-8739-e15cbb2a69c0.png">

I wonder if Releases and Changelog could be kept in sync. Or maybe just keep one or derive one from the other? Duplication being the root of all evil and all that. 

I realize this might not be highest on your list compared to new features and bug fixes but would go a long way to make it easier to use this amazing library.

**Comment by elucash** [comment]
We will probably archive changelog in favor of release notes on tags. Next update will be on 2.9 release, currently we at early access/alpha. thank you!

**Comment by asereda-gs** [comment]
Yes probably we should hide CHANGELOG in favor of github release notes.

Regarding version in immutables docs, the (`immutables.org`) website is statically generated and I have missed upgrading [version variable](https://github.com/immutables/immutables.github.io/blob/src/_config.yml#L7). I wonder if it can be removed altogether (from website) since keeping in sync several versions can be error-prone, as you have pointed out.

**Comment by ncoquelet** [comment]
To sync version in immutables docs, you can generate last release badge with https://shields.io/category/version
For example by using "GitHub release (latest SemVer)" this generate this badge 
![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/immutables/immutables)

And it can take account of pre-releases as discuss in #1251

**Comment by elucash** [comment]
good suggestion, thank you! will use it when will get to it
